tests/group_vars/all/all

```diff
@@ -27,3 +27,5 @@ libreswan_compilation_dependencies_trusty:
   - libunbound-dev
   - make
   - pkg-config
+
+gpg_key_server_address: "x-hkp://pool.sks-keyservers.net"
```

Comment:
I opened https://github.com/jlund/streisand/issues/750 to remember that we should figure out a way to not have to do this duplication going forward.